std_msgs/LaserScan scan          # The original laser scan
std_msgs/PoseStamped[] poses     # The pose during each ray of the laserscan